tests/lib/rules/no-useless-computed-key.js

```diff
@@ -23,7 +23,23 @@ ruleTester.run("no-useless-computed-key", rule, {
         "({ 'a': 0, b(){} })",
         "({ [x]: 0 });",
         "({ a: 0, [b](){} })",
-        "({ ['__proto__']: [] })"
+        "({ ['__proto__']: [] })",
+        { code: "class Foo { 'a'() {} }", options: [{ enforceForClassMembers: true }] },
```

Comment:
It would be good to have here one additional test with an identifier as the key, e.g., `a() {}`, because these are the most common in classes.